draft-pda-protocol.md

```diff
@@ -371,18 +370,35 @@ struct {
 } PAParam;
 
 enum { prio(0), hits(1) } PAProto;
+
+opaque Url<1..2^16-1>;
 ~~~
 
-The `batch_size` field encodes the *batch size*, the number of input shares
-accumulated by each aggregator before emitting its output share.  The `proto`
-field identifies the specific PA protocol in use. The rest of the structure
-contains any protocol-specific parameters that are required.
+* `task`: The PA task.
+* `helper_urls`: The helpers' endpoint URLs.
+* `collector_config`: The HPKE configuration of the collector (described in
+  {{hpke-config}}). [OPEN ISSUE: Maybe the collector's HPKE config should be
+  carried by the collect request?]
+* `batch_size`: The batch size, i.e., the minimum number of reports that are
+  aggregated into an output.
+* `batch_window`: The batch window, i.e., the minimum time difference between
```

Comment:
Are we concerned about skew between collectors and aggregators?

Reply:
We should then (a) note that as a requirement and (b) note what happens if that does not happen.

Reply:
More precisely, I think all that's required for privacy is that each aggregator needs to have a clock that's roughly in sync with true time (within batch_window seconds). Added this to "Pre-conditions".